src/main/java/org/jabref/logic/util/URLUtil.java

```diff
@@ -82,25 +83,37 @@ public static String cleanGoogleSearchURL(String url) {
      * @return true if <c>url</c> contains a valid URL
      */
     public static boolean isURL(String url) {
+        if (!url.matches("^[a-zA-Z]+://.*")) {
+            return false;
+        }
         try {
-            create(url);
+            createUri(url);
             return true;
-        } catch (MalformedURLException | IllegalArgumentException e) {
+        } catch (IllegalArgumentException e) {
             return false;
         }
     }
 
     /**
      * Creates a {@link URL} object from the given string URL.
+     * <p>
+     * If the URL does not contain a protocol (e.g., "example.com"), 
```

Comment:
OMG. Wrong term. Please use the right terms. See https://en.wikipedia.org/wiki/URL#Syntax

`example.com` is not a protocol!!